apps/dashboard/shared/types/blockchains.ts

```diff
@@ -0,0 +1,5 @@
+export enum BlockchainEnum {
+  ETHEREUM = "ETHEREUM",
+}
```

Comment:
why do you need this? you could simply rely on the public client's `getBlockNumber` without checking which chain this is